playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerWrapper.java

```diff
@@ -273,6 +273,10 @@ private DefaultTrackSelector initializeTrackSelector() {
     }
 
     private void preparePlayer(@NonNull PKMediaSourceConfig sourceConfig) {
+        if (this.sourceConfig != null && this.sourceConfig.mediaSource.getMediaFormat() == PKMediaFormat.udp) {
+            player.seekTo(0);
```

Comment:
this is for seek to live edge?

Reply:
no the r seek to live edge is done automatically once media is defined as live

I have updated the logic so now it is more readable and achieve the same goal that playback after change media from udp to hls/dash will make player to return the correct position